charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/blackbox-exporter.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: service-check
```

Comment:
Can we rename this thing to make more clear what it does? "service check" could mean anything. What about something more expressive like "connectivity-checker" or "apiserver-connectivity-checker" or sth in this direction?

Reply:
Renamed to apiserver-connectivity-check